server/views.py

```diff
@@ -18,10 +19,11 @@
 from django.urls import reverse
 
 # Create your views here.
+logger = logging.getLogger(__name__)
 
 ##clean up old requests
 def cleanup():
-    how_many_days = 7
+    how_many_days = 180
```

Comment:
Why are we changing this? I don’t think this should be this long. Doesn’t seem related to the PR.

Reply:
Doesn't need to be. I gotta stop working all night :')